pkg/network/ebpf/c/protocols/http2/decoding-common.h

```diff
@@ -161,4 +161,11 @@ static __always_inline void reset_frame(http2_frame_t *out) {
     *out = (http2_frame_t){ 0 };
 }
 
+// check_frame_split checks if the frame is split into multiple tcp payloads, if so, it increments the split counter.
```

Comment:
```suggestion
// checks if the frame is split into multiple tcp payloads, if so, it increments the split counter.
```

Reply:
shouldn't you check that only for interesting frame?

Reply:
changed it to be included only in interesting frames